apps/cookbook/src/app/showcase/form-field-showcase/form-field-showcase.component.html

```diff
@@ -56,6 +56,12 @@ <h3>Character counter</h3>
 </cookbook-example-viewer>
 
 <h3>Numeric</h3>
+<p>
+  The numeric input restricts the input field to allow only numbers. Chrome completely restricts the
```

Comment:
We could write something like this instead to further emphasize that it is a validation step and that validation happens at different times depending on the browser.
```suggestion
  The numeric input includes built in validation for rejecting non-numerical user input. For example Chrome completely restricts the
```